extensions/rant_patrol.py

```diff
@@ -2,42 +2,29 @@
 import hikari, lightbulb
 import os
 
-plugin = lightbulb.Plugin("rantpatrol")
-
-"""
-Bot deletes improperly formatted rants or vents
-Message must start with 'rant: ' or 'vent: ' followed by literally any alphanumeric text. Lowercase only.
-"""
-
-# helpers
+plugin = lightbulb.Plugin("Rantpatrol")
 
+RANT_REGEX = re.compile("^[^ ]*(rant|vent) *:", flags=re.IGNORECASE)
```

Comment:
No anti/co prefix?

Reply:
Oh I see. Very nice

Reply:
For the purposes of detecting a ranting/venting attempt, any prefix is accepted `[^ ]*`. The next regex then requires that the prefix (if present) be one of an approved set.

Reply:
Then why not remove the `^` and just match anywhere in the message? I forget if newlines are captured by `[^ ]`

Reply:
The message `I wish people would remember to prefix their messages with rant: properly` should not be detected as a rant